test/cpp/test_replication.cpp

```diff
@@ -57,7 +57,7 @@ void TestSingleReplication(
 
   std::vector<std::vector<torch_xla::runtime::ComputationClient::DataPtr>>
       results(device_strings.size());
-  torch_xla::runtime::util::MultiWait mwait(device_strings.size());
+  absl::BlockingCounter mwait(device_strings.size());
```

Comment:
let's rename it to `bc` or something, `mwait` would be confusing if the type is not `MultiWait`.